README.rst

```diff
@@ -85,12 +85,12 @@ After installation the user should be able to import the ``mayavi`` visualizatio
 Directory structure
 -------------------
 
-There are four subpackages:

```

Comment:
visualisation with a ("z") in example above (in line 84),  "from simphony.visualization import mayavi_tools"